tests/NuGetUtility.Test/LicenseValidator/LicenseValidatorTest.cs

```diff
@@ -668,6 +722,66 @@ private IPackageMetadata SetupPackage(string packageId, INuGetVersion packageVer
                     })
                     .Using(new LicenseValidationResultValueEqualityComparer()));
         }
+
+        [Test]
+        [ExtendedAutoData(typeof(NuGetVersionBuilder))]
+        public async Task ValidatingLicenses_ShouldContainCopyright(
+            string packageId,
+            INuGetVersion packageVersion,
+            string copyrigth)
+        {
+            _uut = new NuGetUtility.LicenseValidator.LicenseValidator(_licenseMapping,
+                Array.Empty<string>(),
+                _fileDownloader,
+                _ignoredLicenses.Append(packageId).ToArray());
+
+            IPackageMetadata package = SetupPackageWithCopyright(packageId, packageVersion, copyrigth);
+
+            IEnumerable<LicenseValidationResult> result = await _uut.Validate(LicenseValidatorTest.CreateInput(package, _context));
+
+            Assert.That(result,
+                Is.EquivalentTo(new[]
+                    {
+                        new LicenseValidationResult(packageId,
+                            packageVersion,
+                            _projectUrl.ToString(),
+                            null,
+                            copyrigth,
+                            null,
+                            LicenseInformationOrigin.Ignored)
+                    })
+                    .Using(new LicenseValidationResultValueEqualityComparer()));
+        }
+
+        [Test]
+        [ExtendedAutoData(typeof(NuGetVersionBuilder))]
+        public async Task ValidatingLicenses_ShouldContainAuthors(
+            string packageId,
+            INuGetVersion packageVersion,
+            string authors)
+        {
+            _uut = new NuGetUtility.LicenseValidator.LicenseValidator(_licenseMapping,
+                Array.Empty<string>(),
+                _fileDownloader,
+                _ignoredLicenses.Append(packageId).ToArray());
+
+            IPackageMetadata package = SetupPackageWithAuthors(packageId, packageVersion, authors);
+
+            IEnumerable<LicenseValidationResult> result = await _uut.Validate(LicenseValidatorTest.CreateInput(package, _context));
+
+            Assert.That(result,
+                Is.EquivalentTo(new[]
+                    {
+                        new LicenseValidationResult(packageId,
+                            packageVersion,
+                            _projectUrl.ToString(),
+                            null,
+                            authors,
+                            null,
+                            LicenseInformationOrigin.Ignored)
+                    })
+                    .Using(new LicenseValidationResultValueEqualityComparer()));
+        }
```

Comment:
I'm missing a test with multiple authors here.... With AutoData you should be able to easily create an array of strings